web/src/main/java/com/google/sps/servlets/SaveStockServlet.java

```diff
@@ -45,22 +49,57 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
     KeyFactory keyFactory = datastore.newKeyFactory().setKind("Stock");
+    ArrayList<String> tikkers = new ArrayList<String>();
+    for (int i = 0; i < tik.size(); i++) {
+      String tikk = tik.get(i).text();
+      tikkers.add(tikk);
+    }
+
 
     for (int i = 0; i < tik.size(); i++) {
+      Key myTiks = datastore.newKeyFactory().setKind("Stock").newKey(tikkers.get(i));
 
-      String tikk = tik.get(i).text();
       String pricee = price.get(i).text().replaceAll("[\\\\$,]", "");
```

Comment:
Sorry can you change this to just "price" or something descriptive. Nobody is going to understand what "pricee" means.